.github/workflows/main.yml

```diff
@@ -46,9 +46,14 @@ jobs:
 
       - name: Run appimage in fresh L4T instance
         run: |
-          make image-test
-          make test-package
+          make image-test TARGET=jetson
+          make test-package TARGET=jetson
       
+      - name: Run appimage in fresh debian instance
+        run: |
+          make image-test TARGET=pi
+          make test-package TARGET=pi
+
```

Comment:
out of scope for this PR, but given that this change is happening in both deploy.yml and main.yml, you may be able to merge these

Reply:
@abe-winter do yo mean merging the appimage test steps? or merging deploy.yaml and main.yaml workflows?

Reply:
the latter, merging the files

not saying definitely do this, but if the files are similar enough they can be combined